SystemDocumentation/CodeAccess.txt

```diff
@@ -0,0 +1,61 @@
+This document will give information about our groups code for perusal and printing.
+
+To access our groups code please unzip the POS.zip file and all of the files in here will be explained below.
+
+Below this will be a brief explanation of each file.
+
+For easy instructions on printing the files in the
+root directory you need to print would be:
+Database.py, DatabaseUrlGenerator.py, DatabaseUtils.py, DbConnection.py, UserLoginPackage.py, main.py, script.py
+
+In the templates/ folder for printing:
+inventory.html, Managerpage.html, POS.html, inventory.html, transactions.html
+
+Thats our whole code base, but below are short descriptions of the files. 
+
+In the root directory of github you will find, we will explain what these files are under them. This is where the bulk of our code is.
+- Please ignore this file it just allows us to run tests for our code
+Database.py
+- This is the database class we use to access the database, please visit the system documentation for more information
```

Comment:
this is the system documentation...... Not sure why you would point him to the same file.